src/main/java/org/opentripplanner/routing/alertpatch/TransitAlert.java

```diff
@@ -191,6 +191,19 @@ public Instant getEffectiveEndDate() {
       .orElse(null);
   }
 
+  /**
+   * Checks if the alert has a NO_SERVICE alert active at the requested time.
+   * @param instant
+   * @return
+   */
+  public boolean noServiceOn(Instant instant) {
```

Comment:
Sorry, I think in English "no service at instant" is more correct. Can you rename the method?